README.md

```diff
@@ -1 +1,230 @@
+
+## Implementation
+
+As the first step, you can build a Ballerina service that gives an UUID as the output. You can simply create a file `uuid_service.bal` and add the following content to the service code.
+
+```ballerina
+import ballerina/http;
+import ballerina/system;
+
+endpoint http:Listener uuid_ep {
+    port:8080
+};
+
+@http:ServiceConfig {
+    basePath:"/"
+}
+service<http:Service> uuid_service bind uuid_ep {
+
+    @http:ResourceConfig {
+        path:"/"
+    }
+    gen_uuid(endpoint outboundEP, http:Request request) {
+        _ = outboundEP->respond(system:uuid());
+    }
+
+}
+```
+
+Now you can add the Kubernetes annotations that are required to generate the Kubernetes deployment artifacts. 
+
+```ballerina
+import ballerina/http;
+import ballerina/system;
+import ballerinax/kubernetes;
+import ballerinax/docker;
+
+@kubernetes:Service {
+    name:"uuid-gen", 
+    serviceType:"LoadBalancer",
+    port:80
+}
+endpoint http:Listener uuid_ep {
+    port:8080
+};
+
+@kubernetes:Deployment {
+    enableLiveness:true,
+    image:"<username>/uuid-gen:latest",
+    push:true,
+    username:"<username>",
+    password:"<password>"
+}
+@http:ServiceConfig {
+    basePath:"/"
+}
+service<http:Service> uuid_service bind uuid_ep {
+
+    @http:ResourceConfig {
+        path:"/"
+    }
+    gen_uuid(endpoint outboundEP, http:Request request) {
+        _ = outboundEP->respond(system:uuid());
+    }
+
+}
+```
+We will be building a docker image here, and publishing it to Docker Hub. This is required, since we cannot simply have the docker image in the local registry, and run the Kubernetes applicates in AKS, where it needs to have access to the docker image in a globally accessible location. For this, an image name should be given in the format $username/$image_name in the "image" property, and "username" and "password" properties needs to contain the Docker Hub account username and password respectively. The property "push" is set to "true" to signal the build process to push the build docker image to Docker Hub.
+
+You can build the Ballerina service using `$ ballerina build uuid_service.bal`. You should be able to see the following output. 
+
+```bash
+$ ballerina build uuid_service.bal
+Compiling source
+    uuid_service.bal
+
+Generating executable
+    uuid_service.balx
+	@kubernetes:Service 			 - complete 1/1
+	@kubernetes:Deployment 			 - complete 1/1
+	@kubernetes:Docker 			 - complete 3/3 
+	@kubernetes:Helm 			 - complete 1/1
+
+	Run the following command to deploy the Kubernetes artifacts: 
+	kubectl apply -f /home/laf/dev/uuid-service/kubernetes/
+
+	Run the following command to install the application using Helm: 
+	helm install --name uuid-service-deployment /home/laf/dev/uuid-service/kubernetes/uuid-service-deployment
+```
+
+After the build is done, the docker image would have been created, and pushed to Docker Hub, and also the Kubernetes deployment artifacts would be generated as well.
```

Comment:
Change the tense to present so that it reads similar to the following:

"After the build is complete, the Docker image is created and pushed to Docker Hub. The Kubernetes deployment artifacts are generated as well."

Reply:
Fixed, thanks!.